afbirez/settings/base.py

```diff
@@ -95,3 +95,5 @@
 )
 
 DEFAULT_FROM_EMAIL='catherine.devlin@gsa.gov'
```

Comment:
@catherinedevlin Could we replace this with a dummy address, something like sbirez@gsa.gov? Would those emails be sent?